resources-generator/src/main/kotlin/dev/icerock/gradle/generator/FontsGenerator.kt

```diff
@@ -70,7 +73,9 @@ abstract class FontsGenerator(
 
     @Suppress("SpreadOperator")
     private fun generateFontFamilySpec(
+        //like anastasia
         familyName: String,
+        //key like bold to filename like anastasia-bold.ttf
         fontStyleFiles: List<Pair<String, String>>
```

Comment:
here will be `List<Pair<String, File>>`

Reply:
and in `getPropertyInitializer(fileName)` we can pass `file.nameWithoutExtension`